mod/calculated_metrics/lib/formula/ruby.rb

```diff
@@ -8,7 +8,7 @@ module Formula
   # The formula may only consist of numbers and the symbols and functions
   # listed in SYMBOLS and FUNCTIONS
   class Ruby < NestFormula
-    SYMBOLS = %w{+ - ( ) [ ] . * , / || &&}.freeze
+    SYMBOLS = %w[+ - ( ) \[ \] . * , / || && { } ].freeze
```

Comment:
Do `{` and `}` have the same meaning in Wolfram?

Reply:
No, but now they have in Ruby the same meaning the have in Wolfram.